docs/source/getting-started/content/assembly_common-tasks.adoc

```diff
@@ -2,3 +2,4 @@
 = Common tasks
 
 include::proc_starting-monitoring-alerting-telemetry.adoc[leveloffset=+1]
+include::proc_starting-codeready-containers-with-proxy.adoc[leveloffset=+1]
```

Comment:
Honestly, I don't like the inclusion of a "common tasks" section to the docs. This feels like a random collection of miscellaneous tasks, and that's far from ideal.

I don't think this needs to be changed in this PR, however -- we can keep it here and get the docs merged in. For the future, though, I'd like to reconsider this entire section and start writing more dedicated sections of the docs to cover what's being included here.